kpi/kpi.model.lkml

```diff
@@ -26,7 +26,7 @@ explore: firefox_desktop_usage_2021 {
     type: cross
     relationship: many_to_one
   }
-  hidden: no
```

Comment:
hiding the 2021 KPI related explores from the UI. I'm not removing them because I don't want the old dashboards to break.